cleanlab/experimental/tf_nlp.py

```diff
@@ -0,0 +1,129 @@
+"""
+A Scikeras classifier (adapted for huggingface models) which
```

Comment:
"adapted for huggingface models"
->
"adapted for fine-tuning HuggingFace pretrained Transformer models"